From: Chad Dupuis <cdupuis@redhat.com>
Date: Wed, 22 Sep 2010 20:11:10 -0400
Subject: [net] netxen: update version 4.0.74
Message-id: <20100922201110.2527.25962.sendpatchset@localhost.localdomain>
Patchwork-id: 28346
O-Subject: [RHEL 5.6 PATCH 3/6] netxen: update version 4.0.74
Bugzilla: 625079
RH-Acked-by: David S. Miller <davem@redhat.com>

Bugzilla
--------

Bug 625079 (https://bugzilla.redhat.com/show_bug.cgi?id=625079)

Upstream Status
---------------

net-2.6 commit id 772806bbbc5aa4ddf5ac0de57c1ee7c0ef94e490

Description
-----------

>From c28fc6f0946ccdc6fe87807bbc9fd947d36a0b7e Mon Sep 17 00:00:00 2001
From: Amit Kumar Salecha <amit.salecha@qlogic.com>
Date: Tue, 17 Aug 2010 20:51:52 +0000
Subject: [PATCH] netxen: update version 4.0.74

Signed-off-by: Amit Kumar Salecha <amit.salecha@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/drivers/net/netxen/netxen_nic.h b/drivers/net/netxen/netxen_nic.h
index 503803c..f93b706 100644
--- a/drivers/net/netxen/netxen_nic.h
+++ b/drivers/net/netxen/netxen_nic.h
@@ -54,8 +54,8 @@
 
 #define _NETXEN_NIC_LINUX_MAJOR 4
 #define _NETXEN_NIC_LINUX_MINOR 0
-#define _NETXEN_NIC_LINUX_SUBVERSION 73
-#define NETXEN_NIC_LINUX_VERSIONID  "4.0.73"
+#define _NETXEN_NIC_LINUX_SUBVERSION 74
+#define NETXEN_NIC_LINUX_VERSIONID  "4.0.74"
 
 #define NETXEN_VERSION_CODE(a, b, c)	(((a) << 24) + ((b) << 16) + (c))
 #define _major(v)	(((v) >> 24) & 0xff)
